lib/presentation/widget/drawer_tiles.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+
+import '../../core/util/text_styles/custom_text_style.dart';
+
+class DrawerTiles extends StatelessWidget {
```

Comment:
Change the name to singular, as this returns a single tile.

Reply:
done